test/pkg/integration/integration.go

```diff
@@ -501,3 +503,27 @@ func isPodReady(s *corev1.PodStatus) bool {
 
 	return false
 }
+
+func FindVolumeSnapshot(instanceID, namespace string, client klient.Client) (string, string, error) {
+	vsClient, err := volumesnapshotv1.NewForConfig(client.RESTConfig())
+	if err != nil {
+		return "", "", err
+	}
+
+	volumeSnapshots, err := vsClient.SnapshotV1().VolumeSnapshots(namespace).List(context.Background(), metav1.ListOptions{
```

Comment:
How about getting `ctx` from the caller of `FindVolumeSnaphot`?

Reply:
Nice!